migrations/20210329072840-create-user.js

```diff
@@ -1,51 +1,37 @@
-'use strict';
-
 module.exports = {
   up: async (queryInterface, Sequelize) => {
     await queryInterface.createTable('Users', {
-      
       id: {
         allowNull: false,
         autoIncrement: true,
         primaryKey: true,
-        type: Sequelize.INTEGER
+        type: Sequelize.INTEGER,
       },
-      
       name: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
       },
-      
       email: {
         type: Sequelize.STRING,
         allowNull: false,
       },
-      
       password: {
         type: Sequelize.STRING,
         allowNull: false,
       },
-      
       dob: {
-        type: Sequelize.DATE
-      },
-      
-      age: {
-         type: Sequelize.INTEGER 
+        type: Sequelize.DATEONLY,
       },
-      
       createdAt: {
         allowNull: false,
-        type: Sequelize.DATE
+        type: Sequelize.DATE,
       },
-      
       updatedAt: {
         allowNull: false,
-        type: Sequelize.DATEONLY
-      }
-
+        type: Sequelize.DATEONLY,
+      },
```

Comment:
Add A hook to remove all exercise/workout and other data when a user is deleted. 
See Hooks